src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star__blok">
+      <div class="star__line stars--0">
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--1">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--2">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--3">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--pasive"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--4">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--pasive"></div>
+      </div>
+
+      <div class="star__line stars--5">
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
+        <div class="srar__style star--active"></div>
```

Comment:
Correct the typo 'srar__style' to 'star__style' to ensure the intended styles are applied correctly.